### PR TITLE
Organize Debug Commands into CLI Command Group

### PR DESCRIPTION
**Pull request by glenn-sorrentino**
## Changes

- Encapsulated the `list_keys` function within a Flask CLI command group named `debug`. This helps organize debugging-related commands and keeps the main command namespace clean.

## Why

- Improves discoverability and organization of debugging commands without cluttering the primary command space.

### CODE REVIEW

File: hushline/__init__.py [hunk 1]
```diff
@@ -54,6 +56,19 @@ def handle_rate_limit_exceeded(e):
     for module in [admin, settings]:
         app.register_blueprint(module.create_blueprint())
 
+    # Correctly initialize the debug_cli AppGroup before defining commands
+    debug_cli = AppGroup("debug", help="Debugging related commands")
```
Comment (brassy-endomorph) [inline hushline/__init__.py:60]: This line plus the `app.cli.add_command(debug_cli)` can just be done like this:

```python
@app.cli.group(help="Debugging related commands")
def debug() -> None:
    pass
```

This is how the docs recommend it, and it seems to be preferrable, but this is just a nit.

**Review by brassy-endomorph — APPROVED** [review]

File: hushline/__init__.py (hunk 1 above)
Comment (glenn-sorrentino) [reply hushline/__init__.py]: New update pushed

**Comment by brassy-endomorph** [comment]
Ok, one last comment that I'm gonna refer to as "`git` discipline" but we should avoid merge commits from the base branch and should instead rebase. And to the extend possible we should interactively rebase to squash fixups into their relevant commits. This makes it so much easier to track where a bug came from, and having a clean git history means that all the tooling for showing history and branches actually represents things a bit better. This is an easy change to make for this PR. I think the steps are:

- `git reset HEAD^ --hard` to remove the merge
- `git rebase -i 6d6c1f5^` and then make `b8bcf83` a fixup on top of `6d6c1f5`
- Rebase your branch on main if there's merge conficts, but there shouldn't be?
- Force push

**Comment by glenn-sorrentino** [comment]
> Ok, one last comment that I'm gonna refer to as "`git` discipline" but we should avoid merge commits from the base branch and should instead rebase. And to the extend possible we should interactively rebase to squash fixups into their relevant commits. This makes it so much easier to track where a bug came from, and having a clean git history means that all the tooling for showing history and branches actually represents things a bit better. This is an easy change to make for this PR. I think the steps are:
> 
>     * `git reset HEAD^ --hard` to remove the merge
> 
>     * `git rebase -i 6d6c1f5^` and then make `b8bcf83` a fixup on top of `6d6c1f5`
> 
>     * Rebase your branch on main if there's merge conficts, but there shouldn't be?
> 
>     * Force push

Done. Thanks for the guidance!

**Comment by glenn-sorrentino** [comment]
Closes #280

**Pull request merged**